data-migration/1.x-to-2.x/README.md

```diff
@@ -0,0 +1,4 @@
+# Updating data from 1.x to 2.x
+
+See CHANGELOG.md

```

Comment:
```suggestion
See [../CHANGELOG.md](../CHANGELOG.md#200)

```

Reply:
I moved the instructions that used to be in CHANGELOG to here and added a link to here from the CHANGELOG

Reply:
That also works for me